hack/keepalived-docker/manage-keepalived.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+export assignedIP="$PROVISIONING_IP/$PROVISIONING_CIDR"
+export interface=$PROVISIONING_INTERFACE
```

Comment:
adding a default here could be nice

Reply:
$PROVISIONING_IP, $PROVISIONING_CIDR and $PROVISIONING_INTERFACE already have default values, I believe ....

Reply:
it's in case you do not mount the configmap or anything (for example when running in plain docker, when locally)

Reply:
We can write in the documentation in that case that,  we need to export $PROVISIONING_IP, $PROVISIONING_CIDR and $PROVISIONING_INTERFACE variables.